Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -1516,46 +1516,44 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
     const actorIsIdentity = actor.getIsIdentity();
     const actMats = actorIsIdentity ? null : model.openGLActor.getKeyMatrices();
```

Comment:
I feel that if `actorIsIdentity` is false, setting `actMats` to `{mcwc: null, normalMatrix: null}`  would simplify the code